client/src/document/toolbar/edit-actions.tsx

```diff
@@ -71,6 +71,8 @@ export function EditActions({ folder }: { folder: string }) {
     }
   }
 
+  const params = useParams();
```

Comment:
I know you're just moving this from one place to another but let's use this notation instead:
```suggestion
  const { "*": slug } = useParams();
```
Then, ...